src/sentry/tasks/servicehooks.py

```diff
@@ -1,13 +1,98 @@
 from __future__ import absolute_import, print_function
 
 import six
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 from time import time
 
+from sentry.api.serializers import serialize, app_platform_event
 from sentry.http import safe_urlopen
+from sentry.models import Group, SentryAppInstallation, ServiceHook
 from sentry.tasks.base import instrumented_task
 from sentry.utils import json
 from sentry.utils.http import absolute_uri
 
+# This is an extra, explicit, measure to ensure we only send events for
+# resource changes we deem necessary.
+ALLOWED_ACTIONS = (
+    'issue.created',
+)
+
+# We call some models by a different name, publically, than their class name.
+# For example the model Group is called "Issue" in the UI. We want the Service
+# Hook events to match what we externally call these primitives.
+RESOURCE_RENAMES = {
+    'group': 'issue',
+}
+
+
+@receiver(post_save, sender=Group)
+def resource_changed(sender, instance, created, **kwargs):
+    process_resource_change.delay(sender, instance.id, created)
+
+
+@instrumented_task('sentry.tasks.process_resource_change')
+def process_resource_change(sender, instance_id, created):
+    model = sender.__name__.lower()
+    model = RESOURCE_RENAMES.get(model, model)
+
+    instance = sender.objects.get(id=instance_id)
+
+    event = 'created' if created else 'updated'
+    action = u'{}.{}'.format(model, event)
+
+    if action not in ALLOWED_ACTIONS:
+        return
+
+    project = None
+
+    if isinstance(instance, Group):
+        project = instance.project
+
+    if not project:
+        return
+
+    servicehooks = ServiceHook.objects.filter(
+        project_id=project.id,
+    )
+
+    for servicehook in filter(lambda s: action in s.events, servicehooks):
```

Comment:
Can this filtering operation not be done in SQL?

Reply:
We talked about this in Slack for a bit yesterday. The column is an `ArrayField` and our current DB code doesn't support actual Postgresql operations to work with Array fields (like `<column>__contains`).

So we could update that code to support it (more trouble than it's worth, imo), do something in `extra` via raw SQL, or this. A Project will have so few records (after this PR, 3 at most) I think this filter is a bit easier to grok, personally, and should have negligible performance implications.

Reply:
ok. :+1: